ext/crates/sseq/src/coordinates/lattice.rs

```diff
@@ -0,0 +1,244 @@
+use std::cmp::Ordering;
+
+/// Partially ordered type with greatest lower bounds, or meets.
+pub trait MeetSemilattice: PartialOrd<Self> {
```

Comment:
This file seems like it might add an unnecessary abstraction. Do you have a link to a place where this is used? Maybe split it off into a separate PR and merge the rest of everything first?

Reply:
It's really only used outside of `ext-rs` itself. Jason's code uses it to figure out the region it needs to iterate over when computing Massey products